views/utilities.js

```diff
@@ -227,6 +227,40 @@ module.exports  = {
         return requestData;
     },
 
+    // Searches for any previous request to see if this is a duplicate
+    // Returns null if not found, otherwise returns first record
+    findPriorRequest: function(requestData) {
+      // Replacing non-numeric characters with Postgres match operator, adding 
+      // one to the beginning to accommodate country codes
+      var phoneQuery = "%" + requestData.phone.replace(/[^0-9]/g, "%");
+
+      return db.Request.findOne({
+          where: {
+            name: { $ilike: requestData.name },
+            source: requestData.source,
```

Comment:
I actually think that even if the source is different we should still mark it as a duplicate!  I can see a case where someone called 311 and then also made a request by text message with the same information -- in that case I think it should still be marked as a duplicate.  What do you think?

Reply:
I'd agree with that, good point! I'll make that update